tests/test_showcase.py

```diff
@@ -148,10 +150,10 @@ def test_api(showcase):
         isinstance(p, (Assignment, Unassignment, Commit))
         for p in rf_ver1.get_events(direct=False) + rf_ver2.get_events(direct=False)
     )
-    rf_a4, rf_a1, rf_c1 = rf_ver1.get_events(
+    _rf_a4, rf_a1, _rf_c1 = rf_ver1.get_events(
         direct=False
     )  # pylint: disable=unused-variable
-    rf_a3, rf_a2, rf_c2 = rf_ver2.get_events(
+    rf_a3, rf_a2, _rf_c2 = rf_ver2.get_events(
```

Comment:
Yeah I just forgot to remove the pylint warning here (I removed the top file-level disable for `unused-variable` and missed that it was also explicitly disabled on this line). Prefixing any of the variables with `_` fixes the warning, I left the variable naming in place (i.e. `rf_...` -> `_rf_...`) since it may make it easier for anyone reading this test to see what `get_events` is supposed to be returning